src/shared-components/callout/index.tsx

```diff
@@ -5,7 +5,7 @@ import { useTheme } from 'emotion-theming';
 import Style from './style';
 import { COLORS_PROP_TYPES, ThemeColors } from '../../constants';
 
-type CalloutProps = {
+interface CalloutProps {
```

Comment:
Not important for this, but I've found that it's really helpful in other libraries when they always export interfaces like all of these.  That way if I want to create a wrapper or something like that I can just do `React.FC<{myProp1: number, subProps: CalloutProps}>` (that's probably a bad example).